test/models/run_test.exs

```diff
@@ -15,4 +17,12 @@ defmodule Gatherto.RunTest do
     changeset = Run.changeset(%Run{}, @invalid_attrs)
     refute changeset.valid?
   end
+
+  describe "persistance" do
+    test "a run with valid attributes" do
+      Run.changeset(%Run{}, @valid_attrs) |> Repo.insert!
+
+      assert Repo.get_by!(Run, @valid_attrs)
+    end
+  end
```

Comment:
@matt-keibler I added this as I was thinking about the value of having a simple test to make sure that each 'model' could be persisted. This might be overkill because the controller tests may handle this but it seemed to make sense at the time.